admin/aioseop_module_class.php

```diff
@@ -821,11 +821,14 @@ function get_object_labels( $post_objs ) {
 		 *
 		 * @return array
 		 */
-		function get_term_labels( $post_objs ) {
+		function get_term_labels( $post_objs, $prefixed_with_taxonomy = false ) {
```

Comment:
PHPDoc needs to be updated, and noting a change in `since`.

Reply:
done